src/main.rs

```diff
@@ -375,11 +382,10 @@ fn main() {
 			.filter_map(|(name, loc)| match loc {
 				gb_sym_file::Location::Banked(bank, addr) => Some((name, (bank, addr))),
 				_ => None,
-			})
-			.collect()
-	} else {
-		HashMap::new()
-	};
+			});
+		symfile.extend(symbols);
```

Comment:
This change is opinionated, and it is honestly a nitpick. In general, I don't really like having a short `else` clause that follows a long `if` body. However, I think this `else` clause is specifically not visible enough because it supplies important type information.

Reply:
would be better then to declare `symfile` immediately when it is first assigned to, which is here...